docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2102,7 +2107,7 @@ Status Codes:
 
 `POST /exec/(id)/resize`
 
-Resizes the `tty` session used by the `exec` command `id`.
+Resizes the `tty` session used by the `exec` command `id`.  The unit is number of characters.
 This API is valid only if `tty` was specified as part of creating and starting the `exec` command.
```

Comment:
Shouldn't the example below include the `?w=xxx&h=xxx` parameters? (Not your fault, but I just noticed this)